app/controllers/spree/admin/tax_rates_controller.rb

```diff
@@ -14,7 +14,7 @@ def update
       private
 
       def amount_changed?
-        BigDecimal(permitted_resource_params[:amount]) != @tax_rate.amount
+        BigDecimal(permitted_resource_params[:amount].to_s) != @tax_rate.amount
```

Comment:
Why do we want to call `to_s` here?

Reply:
Updated. The `BigDecimal` class was throwing and error here unless it received a string, but I think that issue was down to how the params were being passed in the relevant spec, and not the controller itself.